controller/controller_server/csi_controller_server.py

```diff
@@ -13,28 +10,25 @@
 from controller.array_action import messages
 from controller.array_action.storage_agent import get_agent, detect_array_type
 from controller.common import settings
-from controller.common.csi_logger import get_stdout_logger, set_log_level
+from controller.common.csi_logger import get_stdout_logger
 from controller.common.node_info import NodeIdInfo
 from controller.common.utils import set_current_thread_name
+from controller.controller_server import messages as controller_messages
 from controller.controller_server.errors import ObjectIdError, ValidationException
 from controller.controller_server.exception_handler import handle_common_exceptions, handle_exception, \
     build_error_response
-from controller.controller_server import messages as controller_messages
 from controller.csi_general import csi_pb2
 from controller.csi_general import csi_pb2_grpc
 
 logger = get_stdout_logger()
 
 
-class ControllerServicer(csi_pb2_grpc.ControllerServicer):
+class CSIControllerServicer(csi_pb2_grpc.ControllerServicer):
```

Comment:
I suggest aligning the file name and the class name.

Reply:
I'm not sure it's related to this PR

Reply:
Then why did you rename the class?

Reply:
to distinguish between csi and the upcoming non-csi controller logic

Reply:
Usually, class names go together with the filename.

Reply:
I agree, but since it was already unaligned (if anything, this diff brings it closer to be aligned), I'm not sure it's related to this PR

Reply:
If so, I suggest reverting this change since it's not related to moving out the main function from this file.

Reply:
so you want me to open a separate PR just for this change?

Reply:
I want to align the file name with the class name, but you don't want to do that now because it's not related to the PR.
Thus I suggest not doing things that aren't related to this PR.
If you want to do it in a separate PR, I don't mind.

Reply:
What's the difference? MY suggestion is related to this PR more than it's related to the big replication addition PR.
@ArbelNathan , WDYT?

Reply:
your suggestion is an improvement for a general gap in the project (class-filename inconsistency), which I think should be addressed in its own PR, while this change is meant to make current controller class more specific, to distinguish between CSI and Replication controller classes

Reply:
At the moment there is no replication controller class. No need to distinguish now between them.

Reply:
I think it could be done in this PR but if there is a general gap in the whole project so it could be done on a different ticket.

Reply:
I suggested to do this distinction before the replication addition to reduce the replication diff.
so back to [my question](https://github.com/IBM/ibm-block-csi-driver/pull/355#discussion_r687647059) - would it be better to include this as part of the replication PR?

Reply:
Yes.

Reply:
done